design/authenticators/authn_sut/authn_sut_solution_design.md

```diff
@@ -0,0 +1,433 @@
+# Single Use Token (SUT) Authenticator
+
+## Overview/Introduction
+
+The Single Use Token (SUT) Authenticator is a new authenticator for Conjur that
+will allow users to authenticate to Conjur using a single-use token. A user will
+be able to generate a single use token by providing their username and password
+to a new API endpoint. The user will then be able to use the token to
+authenticate to Conjur.
+
+## Summary of Existing Functionality
+
+There is currently no way to create a single-use token for authenticating to
+Conjur.
+
+## Requirements
+
+The driver behind this effort is to enable integration between Conjur UI and
+PSM. That integration will require a mechanism for generating a single-use token
+that can be used to authenticate to Conjur. The requirements for this effort are
+as follows:
+
+### Feature Spec for Conjur UI-PSM Integration (CNJR-2392)
+
+> Privileged Session Manager (PSM) enables users to log on to remote
+> machines/applications securely through a proxy machine facilitating
+> streamlined and native workflows for the IT admins. Conjur is now going to
+> integrate with PSM providing integrity across Cyberark products and easy
+> access to the Conjur UI via PSM.
+>
+> In order for PSM to connect to the Conjur UI, a new API(s) would be provided
+> that PSM could call to establish the UI session and login the user directly
+> into the main UI dashboard, without showing the initial login screen. More
+> details will be added in the solution design (TBD).
+>
+> The overall UX will remain the same where users will continue to use the
+> existing workflow of logging onto the PVWA, clicking connect on a defined
+> Conjur instance 1 and connecting through PSM straight into the Conjur UI
+> console.
+
+The SUT authenticator will be the mechanism used to authenticate the user to
+Conjur when they click "Connect" on a Conjur instance in PVWA. PSM will generate
```

Comment:
Minor nit: PSM isn't generating the SUT. Conjur generates the SUT and passes it back to PSM.